base/setup/lib/bootsup.c

```diff
@@ -1404,14 +1508,17 @@ InstallVBRToPartition(
                                                SourceRootPath,
                                                DestinationArcPath);
     }
-    /*
-    else if (wcsicmp(FileSystemName, L"EXT2")  == 0 ||
-             wcsicmp(FileSystemName, L"EXT3")  == 0 ||
+    else if (wcsicmp(FileSystemName, L"EXT2")  == 0)
+    {
+        return InstallExt2BootcodeToPartition(SystemRootPath,
+                                               SourceRootPath,
+                                               DestinationArcPath);
+    }
+    else if (wcsicmp(FileSystemName, L"EXT3")  == 0 ||
```

Comment:
just a question oout of curiousity: you are sure that EXT3 is not supported?

Reply:
theoretically it should work. but to be 100% honest, I only focused on ext2 for now. I'll see for ext3 and possibly ext4 later with [Ext4Fsd](https://github.com/bobranten/Ext4Fsd).